### PR TITLE
docs(production): update recipe with AWS deployment

### PR DESCRIPTION
**Pull request by infoxicator**
Added example on how to deploy One App to AWS Elastic Beanstalk

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation (adding or updating documentation)
- [ ] Dependency update

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

### CODE REVIEW

**Review by JAdshead — DISMISSED** [review]

**Review by JAdshead — DISMISSED** [review]

File: docs/recipes/Running-In-Production.md [hunk 1]
```diff
@@ -156,4 +159,64 @@ One App is configured via [environment variables](#). There are a few
 environment variables that are required for One App to start up including the one used to let One App
 know where to fetch a module map from as described above.
 
+### Minimum Required Environment Variables:
+
+In order for One App to start after it has been deployed, the following environment variables must be set before attempting to deploy / start the One App container:
```
Comment (nellyk) [inline docs/recipes/Running-In-Production.md:164]: should we link the env-variables file here?

File: docs/recipes/Running-In-Production.md [hunk 2]
```diff
@@ -156,4 +159,64 @@ One App is configured via [environment variables](#). There are a few
...
+You can deploy the published [One App production image](https://hub.docker.com/r/oneamex/one-app) from Docker Hub to Elastic Beanstalk by following these steps:
+
+1. Go to the AWS console, select Elastic Beanstalk and click on "Create Application".
+2. Give your application a name and select "Docker" as the platform.
+3. Select "Sample Application" and click "Create Application".
+4. Before we can upload and install the One App image to our new environment, we need to add the [Minimum Required Environment Variables](#minimum-required-environment-variables). Click on the "Configuration" of your new environment,
+under the "Software" category click "Edit". Enter all the required environment variables under the "Environment Properties" section.
+
+> Important: Ensure that you have entered a valid `HOLOCRON_MODULE_MAP_URL` that is publicly accessible and that at least the root module specified in `ONE_CLIENT_ROOT_MODULE_NAME` is published; failure to do so, will result in One App crashing and unable to start.
+
```
Comment (nellyk) [inline docs/recipes/Running-In-Production.md:199]: images would be amazing for the step by step. Maybe something we can consider for the future

File: docs/recipes/Running-In-Production.md (hunk 1 above)
Comment (infoxicator) [reply docs/recipes/Running-In-Production.md:164]: It was meant to be in line 185 but the link was going nowhere, fixed...

**Review by nellyk — APPROVED** [review]

**Review by 10xLaCroixDrinker — APPROVED** [review]

File: docs/recipes/Running-In-Production.md [hunk 3]
```diff
@@ -152,8 +155,68 @@ NODE_ENV=production npm run build
...
+  },
+  "Ports": [
+    {
+      "ContainerPort": "3000"
+    }
+  ]
+}
+```
+> Note: You can select the desired version of One App deployed to Docker Hub instead of latest by changing the tag in the image name.
+
```
Comment (Francois-Esquire) [inline docs/recipes/Running-In-Production.md:217]: If we're running in production, should we mention or link to best practices when deploying with AWS?

File: docs/recipes/Running-In-Production.md [hunk 4]
```diff
@@ -152,8 +155,68 @@ NODE_ENV=production npm run build
...
+After the deployment is complete, you can navigate to your application by clicking on "Go to environment". If the application health displays an error, One App might be failing to start due to a missing environment variable or missing configuration. You can request the logs and the console should display the reason why One App is crashing.
```
Comment (Francois-Esquire) [inline docs/recipes/Running-In-Production.md:218]: Would we need to worry about a readiness probe or health check route with AWS?

File: docs/recipes/Running-In-Production.md (hunk 3 above)
Comment (infoxicator) [reply docs/recipes/Running-In-Production.md:217]: @Francois-Esquire link to AWS docs is at the bottom or what link are you referring to?

File: docs/recipes/Running-In-Production.md (hunk 4 above)
Comment (infoxicator) [reply docs/recipes/Running-In-Production.md:218]: That is an extra implementation that is outside of scope, if the pod starts AWS assumes the app is healthy and docker deployment is correct

**Review by Francois-Esquire — APPROVED** [review]

**Pull request merged**